packages/frontend/src/views/account/my-tickets.vue

```diff
@@ -35,8 +35,19 @@
   <input type="checkbox" id="transfer-ownership-modal" class="modal-toggle">
   <label for="transfer-ownership-modal" class="cursor-pointer modal" style="background-color: #00000055;">
     <label class="relative modal-box" for="">
-      <h3 class="text-lg font-bold">Congratulations random Interner user!</h3>
-      <p class="py-4">You've been selected for a chance to get one year of subscription to use Wikipedia for free!</p>
+      <h3 class="text-lg font-bold">Enter Receipt Details</h3>
```

Comment:
มันควรเป็น Enter Username To Transfer ป่าว Receipt มันเกี่ยวกับใบเสร็จ ไม่เกี่ยวกับช่องนี้

Reply:
ใช้คำว่า Enter recipient  To Transfer ได้มั้ยตั้งใจจะพิมพ์ recipient คำนีแค่ ใช้คำผิดเป็น recipient

Reply:
![10111](https://user-images.githubusercontent.com/95273691/160244238-88f82826-0cf6-4db7-bd6e-a5ede05d6d3c.JPG)

Reply:
Enter Recipient Username To Transfer